Symfony/CS/Tests/Fixer/PSR2/MultipleUseFixerTest.php

```diff
@@ -40,21 +40,22 @@ public function provideCases()
             ),
             array(
                 <<<'EOF'
-use Some, Not, PHP, Like, Use, Statement;
```

Comment:
Still, no need to remove the test!
Just split it into 2:
https://3v4l.org/KThCo